torch/_dynamo/utils.py

```diff
@@ -306,16 +307,17 @@ def dynamo_timed(
             chromium_log.log_event_end(
                 phase_name,
                 time.time_ns(),
-                {"cache_stats": get_cache_stats()},
+                {"cache_stats": get_cache_stats(), "fn_name": key},
                 start,
             )
-        chromium_log.log_event_end(
-            key, time.time_ns(), {"cache_stats": get_cache_stats()}, start
-        )
+        else:
+            chromium_log.log_event_end(
+                key, time.time_ns(), {"cache_stats": get_cache_stats()}, start
```

Comment:
This is not related to the PR, but whenever I've looked and perfetto traces, I have always found the cache stats annotation on these events difficult to understand. Is this the stats of caching *in that span*, or is it the cumulative stats everywhere?  I believe it's cumulative, but the way Perfetto renders makes it look like it's for the span only.